src/i18n/de-de/index.js

```diff
@@ -0,0 +1,358 @@
+            too_small: 'Minimalwert für int{ size } ist { max }',
+            too_large_pow2: 'Maximalwert für uint{ size } ist 2^{ size } - 1',
+            too_small_pow2: 'Minimalwert für int{ size } ist -(2^{ size }) + 1',
+            too_large_unsigint: 'Maximalwert für uint{ size } ist 2^{ size } - 1',
+            no_negative_unsigint: 'Wert für uint{ size } darf nicht negativ sein',
+            invalid_signed_integer: 'Ungültiger signed integer',
+            invalid_unsigint: 'Der Eintrag muss einen gültigen unsigned integer enthalten',
+            str_input_placeholder: '["some value", ... , "final value"]',
+            str_input_hint: 'Doppelte Anführungszeichen in Strings müssen escaped werden (\\")',
+            address_placeholder: 'Adresse beginnt mit 0x',
+            address_label: '{ label } (Adresse/Konto)',
+            boolean_array_label: '{ label } (bool[{ size }])',
+        },
+        health: {
+            status: 'Status',
+            checked_at: 'Geprüft am',
+            task: 'Aufgabe',
+            message: 'Nachricht',
+            block_height: 'Block Nummer',
+            latency: 'Latenz',
+            click_to_change_format: 'Zum Ändern des Formats anklicken',
+        },
+        contract_tab: {
+            copy_abi_to_clipboard: 'ABI JSON des Smart Contracts in die Zwischenablage kopieren',
+            enter_amount: 'Wählen Sie Anzahl der Dezimalstellen und geben Sie einen Betrag ein, dieser wird als Funktionsparameter als uint256 erfasst',
+            result: 'Ergebnis',
+            view_transaction: 'Transaktion ansehen',
+            code: 'Code',
+            read: 'Lesen',
+            write: 'Schreiben',
+            amount: 'Anzahl',
+            value: 'Wert',
+            custom_decimals: 'Benutzerdefinierte Dezimalen',
+            custom: 'Benutzerdefiniert',
+            unverified_contract_source: 'Dieser Smart Contract wurde nicht überprüft..',
+            click_here: 'Klicken Sie hier, ',
+            upload_source_files: 'um Quelldateien hochzuladen und diesen Smart Contract zu überprüfen. ' +
+                'Alternativ können Sie mit dem Smart Contract interagieren, indem Sie ein beliebiges ABI verwenden:',
+            use_erc20_abi: 'ERC20 ABI nutzen',
+            use_erc721_abi: 'ERC721 ABI nutzen',
+            abi_from_json: 'ABI aus JSON-Datei',
+            paste_abi_json_here: 'ABI JSON hier einfügen',
+            abi_json_preview: 'ABI JSON Vorschau',
+            provided_abi_invalid: 'Bereitgestellte ABI ist ungültig',
+            provided_json_invalid: 'Bereitgestellte JSON ist ungültig',
+            read_functions: 'Lese-Funktionen',
+            write_functions: 'Schreib-Funktionen',
+            unverified_contract: 'Ungeprüfter Smart Contract',
+        },
+        header: {
+            sign_in: 'Anmelden',
+            sign_out: 'Abmelden',
+            liq_staking: 'Liquid Staking / flexible Anlage',
```

Comment:
@Kersten-CR I do have mixed feelings about keeping the liquid staking alongside the german translation here because it takes so much space from the menu. If possible, I would choose the one that makes more sense to the native german speaker.
![Screenshot 2023-01-30 at 12 12 04](https://user-images.githubusercontent.com/2939354/215584787-785c4359-5da9-4d72-8a1a-ac7521423e01.png)